File: app/src/main/java/intellij/class9/collections/exercise3/java/Main.java
package intellij.class9.collections.exercise3.java;

import java.util.HashSet;

public class Main {
    public static void main(String[] args) {
        HashSet<Integer> conjunto = new HashSet<>();
        conjunto.add(1);
        conjunto.add(5);
        conjunto.add(6);
        conjunto.add(7);
        conjunto.add(8);
        conjunto.add(9);
        conjunto.add(10);
        conjunto.add(4);
        conjunto.add(2);
        conjunto.add(3);

        Prova prova = new Prova();
        prova.somaTotal(conjunto);
    }
}

File: app/src/main/java/intellij/class9/collections/exercise3/kotlin/Main.kt
package intellij.class9.collections.exercise3.kotlin

import java.util.HashSet

fun main(args: Array<String>) {
    Prova().somaTotal(HashSet<Int>().apply {
        add(1)
        add(5)
        add(6)
        add(7)
        add(8)
        add(9)
        add(10)
        add(4)
        add(2)
        add(3)
    })
}